src/Grid/grid.jl

```diff
@@ -1450,3 +1615,89 @@ struct SurfaceOrientationInfo
     #flipped::Bool
     #shift_index::Int
 end
+
+"""
+    InterfaceOrientationInfo(grid::AbstractGrid, this_face::FaceIndex, other_face::FaceIndex)
+
+Orientation information for interfaces. Such an interface can be 
+possibly flipped (i.e. the defining vertex order is reverse to the 
+spanning vertex order) and the vertices can be rotated skewed against each other, and skewed in the case of triangles.
+Take for example the faces
+```
+1           3
+| \\         | \\
+|  \\        |  \\
+| A \\       | B \\ 
+|    \\      |    \\
+2-----3     1-----2  
+```
+which are skewed and rotated against each other by 90° or the faces
+```
+1           3
+| \\         | \\
+|  \\        |  \\
+| A \\       | B \\ 
+|    \\      |    \\
+2-----3     2-----1  
+```
+which are flipped about the centroid vector. Any combination of these can happen. 
+The combination to map other facet to current facet is encoded with
+this data structure via ``\\vec{p}_{\\text{there}} = [T] \\vec{p}_{\\text{here}}`` where ``[T]`` is `InterfaceOrientationInfo.transformation`.
+"""
+struct InterfaceOrientationInfo
+    flipped::Bool
+    transformation::Matrix{Float64}
+end
+
+function InterfaceOrientationInfo(grid::AbstractGrid, this_face::FaceIndex, other_face::FaceIndex)
```

Comment:
If I understand correctly,  face->face transformation is predetermined, so we can make a function that maps quadrature point indices instead of transforming coordinates

Reply:
The permutation can be computed in the constructor though. I think that would be much cheaper compared to i) updating the quadrature points and ii) re-compute the shape values and gradients in every reinit.